src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -410,11 +409,12 @@ private void EnsureSignature()
         /// If the backing field is unknown, set it to null.
         /// </summary>
         /// <remarks>
-        /// This should be called only if we're sure the backing field can't be non-null value.
+        /// This should be called only if we're sure the backing field can't become non-null value if it's not already.
         /// </remarks>
         internal void MarkBackingFieldAsCalculated()
         {
             Interlocked.CompareExchange(ref _lazyBackingFieldSymbol, null, _lazyBackingFieldSymbolSentinel);
+            ((SynthesizedBackingFieldSymbol)_lazyBackingFieldSymbol)?.MarkIsBoundToFieldKeywordAsCalculated();
```

Comment:
No tests currently cover this. I'll either remove it if it's redundant or add a test.

Reply:
> No tests currently cover this. I'll either remove it if it's redundant or add a test.

Let's try getting into a working mode when a change without a proper testing is not presented for a review. If you'd like to make a change that changes a behavior, try starting with a test first.